pages/search.js

```diff
@@ -33,17 +33,16 @@ export default function Search() {
                     </div>
 
                     {results.length > 0 ? (
-                        results.map(({ repoName, repoOwner, projectType, repoLogo, repoOwnerIcon }) => {
+                        results.map(({ repoName, repoOwner, projectType, repoLogo }) => {
                             return (
```

Comment:
i thought i told yall this before but remove the double returns you can just do `=> (` instead of `=> { return (`